UPGRADE-API-1.13.md

```diff
@@ -90,3 +90,19 @@
    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ShippingMethodDocumentationModifier`
    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\PathHiderDocumentationModifier`
    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\AcceptLanguageHeaderDocumentationModifier`
+
+1. All usages of `ApiPlatform\Core\Api\IriConverterInterface` have been switched to its non-deprecated counterpart `ApiPlatform\Api\IriConverterInterface`.
+   Due to that, the constructor and usage in the following classes have been changed accordingly: 
```

Comment:
I do not see that any of the constructors have changed

Reply:
It's an implicit change since the interface is named the same, nevertheless trying to inject a service with the previous interface will now fail.